src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -602,7 +640,7 @@ private static void RunTests(bool persistLogFileToDisk)
 
             IFileSystem mockFileSystem = CreateMockFileSystem(files, specifier, logFileContents, logFileDirectory);
```

Comment:
>mockFileSystem [](start = 24, length = 14)

Ultra-nit naming suggestion: I usually use `mockFileSystem` to refer to the actual mock object `Mock<IFileSystem>`, and just plain `fileSystem` to hold `mockFileSystem.Object`. Note how within the method `CreateMockFileSystem`, `mockFileSystem` refers to the mock object, but here it refers to the "actual" file system.

Reply:
Got it, thanks. As noted, am looking to conform to your general patterns in testing of this kind, i like the conventions you've set.

---
In reply to: [316895841](https://github.com/microsoft/sarif-sdk/pull/1658#discussion_r316895841) [](ancestors = 316895841)